docs/yaml/yaml.go

```diff
@@ -123,7 +123,7 @@ func GenYamlCustom(cmd *cobra.Command, w io.Writer) error {
 			cliDoc.Kubernetes = true
 		}
 		if _, ok := curr.Annotations["swarm"]; ok && !cliDoc.Swarm {
-			cliDoc.Kubernetes = true
+			cliDoc.Swarm = true
```

Comment:
Something still looks weird here.

It shouldn't need to check `!cliDoc.Swarm` because if it's true, setting it to true again doesn't hurt. Is this just an unnecessary statement that can be removed?

I guess all the checks above are the same, so maybe it was just copied?

Reply:
Yes, was looking at that as well, but didn't want to change without going through the whole logic. I can look at it for a follow-up 👍